hummingbot/connector/exchange/ndax/ndax_order_book_tracker.py

```diff
@@ -52,17 +51,6 @@ def exchange_name(self) -> str:
         """
         return CONSTANTS.EXCHANGE_NAME
 
-    async def _init_order_books(self):
```

Comment:
The function `_init_order_books()` was overwritten in NdaxOrderBookTracker on purpose. Since the L2 order book event NDAX provides in its websocket API returns as the first response the complete order book status, we want to use that information to create the order book. If we don't overwrite the `_init_order_books()` function then the traditional order book initialization is performed: by making a request to the REST API. If this happens we are requesting two times the same information from the server (and consuming limit). It is true that initializing the order book might take a couple more seconds (because the order book data stream has to be initialized and started), but that is not really a problem for the strategy performance.

Reply:
Thank you for the clarification. I have re-introduced the delayed initialization, adjusting the point at which `_order_books_initialized` is set to reflect that

Reply:
After further investigation and discussions with @aarmoa, function was removed once again to ensure the validity of the `self._order_books_initialized event.